config.lua

```diff
@@ -11,14 +11,16 @@ function get_config()
     -- Load the configuration file
     local conf_file = assert(io.open(conf_path, "r"), "Configuration file is missing")
     local conf = json.decode(conf_file:read("*all"))
-
+    conf_file:close()
```

Comment:
I should check if con_file is not `nil`

Reply:
eeeh bumpity bump ? apart from this comment this PR looks like easy to merge?

Reply:
```suggestion
    if conf_file ~= nil then
        conf_file:close()
    end
```

Reply:
Indeed, I think it should works™ ( suggestion not tested)